app/furniture/marketplace/cart.rb

```diff
@@ -1,30 +1,27 @@
 # frozen_string_literal: true
 
 class Marketplace
-  class Cart < ApplicationRecord
+  class Cart < Record
     self.table_name = "marketplace_carts"
-    include WithinLocation
     self.location_parent = :marketplace
 
+    default_scope { where(status: :pre_checkout) }
+
     belongs_to :marketplace, inverse_of: :carts
-    belongs_to :shopper, inverse_of: :carts
     delegate :space, :room, to: :marketplace
+
+    belongs_to :shopper, inverse_of: :carts
+
     has_many :cart_products, dependent: :destroy, inverse_of: :cart
     has_many :products, through: :cart_products, inverse_of: :carts
-    has_one :checkout, inverse_of: :cart
-    has_one :order, inverse_of: :cart
 
     enum status: {
       pre_checkout: "pre_checkout",
-      checked_out: "check_out"
+      paid: "paid"
     }
 
-    def self.model_name
-      @_model_name ||= ActiveModel::Name.new(self, ::Marketplace)
-    end
```

Comment:
I pulled this up into `Marketplace::Record` so all the `Marketplace` domain-objects don't need to define their own model-name.